code/datums/gamemodes/sensor_capture.dm

```diff
@@ -21,6 +21,11 @@
 	. = ..()
 	addtimer(CALLBACK(src, /datum/game_mode/combat_patrol.proc/set_game_timer), SSticker.round_start_time + shutters_drop_time) //game end timer will start ticking down on shutter drop
 
+/datum/game_mode/combat_patrol/sensor_capture/game_end_countdown()
+	if(game_timer == "paused")
```

Comment:
why isnt this null
or at minimum a define

Reply:
Different message for round end vs paused. I'll make a define though